pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -383,22 +393,25 @@ func TestAddPodWillConfirm(t *testing.T) {
 
 	for i, tt := range tests {
 		t.Run(fmt.Sprintf("case_%d", i), func(t *testing.T) {
```

Comment:
In a follow up, we can remove the test table, as there is only one test. Or add more tests and a name.

And similar for other cases below.

Reply:
Yes. I will open a new PR to follow up.